packages/cli/src/command-helpers/download.ts

```diff
@@ -0,0 +1,13 @@
+import { createWriteStream } from 'fs';
+import { http } from 'gluegun';
+
+export async function downloadFile(fileUrl: string, outputLocationPath: string) {
+  const writer = createWriteStream(outputLocationPath);
+  const api = http.create({
+    baseURL: fileUrl,
+  });
+  return api.get('', {}, { responseType: 'stream' }).then((response: any) => {
+    response.data.pipe(writer);
+    return Promise.resolve(outputLocationPath);
+  });
+}
```

Comment:
I would use `fetch`, we maintain an internal wrapper that also passes some headers that can let origin servers know where the traffic is coming from.

https://github.com/graphprotocol/graph-tooling/blob/1f8d86f10ceee6f8e34ba942cfa76a37f1f44a42/packages/cli/src/fetch.ts#L5

Reply:
Have now resolved this to use the wrapped fetch